src/Compilers/Core/Portable/CorLightup.cs

```diff
@@ -48,6 +48,138 @@ private static class CultureTypes
                     return null;
                 }
             }
+
+            private static class _Assembly
+            {
+                internal static readonly Type Type = typeof(Assembly);
+
+                internal static readonly Func<byte[], Assembly> Load_bytes = Type
+                    .GetTypeInfo()
+                    .GetDeclaredMethod("Load", typeof(byte[]))
+                    .CreateDelegate<Func<byte[], Assembly>>();
```

Comment:
I've followed general pattern we use. I'll keep it as is for now. Andy is working on moving Compiler on CoreCLR portable contracts. once that's done we can clean up all the reflection code.